src/main/java/me/mrCookieSlime/Slimefun/SlimefunPlugin.java

```diff
@@ -9,6 +9,7 @@
 import java.util.logging.Level;
 import java.util.stream.Collectors;
 
+import io.github.thebusybiscuit.slimefun4.implementation.listeners.*;
```

Comment:
Please do not use wildcard imports.

Reply:
I thought I fixed that, I must have accidentally used the auto formater.
Fixed in my latest Commit.